membership-attribute-service/app/controllers/HealthCheckController.scala

```diff
@@ -16,14 +17,25 @@ class BoolTest(name: String, exec: () => Boolean) extends Test {
   override def ok = exec()
 }
 
-class HealthCheckController extends Results {
-  lazy val zuora = NormalTouchpointComponents.zuoraService
-  lazy val salesforce = NormalTouchpointComponents.contactRepo.salesforce
+class HealthCheckController extends Results with LazyLogging {
```

Comment:
Do we gain anything from using lazy vals here?

Reply:
😨